client/src/components/applicants/Applicants.js

```diff
@@ -84,9 +91,12 @@ export class Applicants extends Component {
           {applicants ? 'All applicants' : jobPosting.title}
         </div>
         {!adminView &&
-          <Link to='/position/new' style={{ textDecoration: 'none' }}>
-            <button className='applicants__button' onClick={this.handleCopyStages}>
-              Copy Templates
+          <Link
+            to={{ pathname: '/position/new', state: { mode: 'copy' } }}
+            style={{ textDecoration: 'none' }}
+          >
+            <button className='applicants__button' onClick={this.handleCopyJobPosting}>
+              Copy Position
```

Comment:
Nitpicking, but could 'Copy as template' be a more intuitive label for the button?